src/main/webapp/result-forms/route.js

```diff
@@ -110,7 +111,7 @@ export const Route = props => {
   }
 
   if (error) {
-    return <div>Error</div>
+    return <ErrorMessage />
```

Comment:
Any particular reason why this `ErrorMessage` doesn't have any additional info or has the ability to refetch?

Reply:
I'll switch these to use the retry snackbar component to be consistent with the workspaces route